src/iiif.py

```diff
@@ -49,7 +49,9 @@ def iiif_info(msid, filename):
         'iiif_info_url': iiif_info_url(msid, filename)
     }
     try:
-        resp = utils.requests_get(iiif_info_url(msid, filename))
+        url = iiif_info_url(msid, filename)
+        LOG.info("Loading IIIF info URL: %s", url)
```

Comment:
this should not change, and should not be the CDN since it's loaded from inside the VPC